.vitest/src/instances.ts

```diff
@@ -10,6 +10,17 @@ import { split } from "../../aa-sdk/core/src/transport/split";
 import { poolId, rundlerBinaryPath } from "./constants";
 import { paymasterTransport } from "./paymaster/transport";
 
+export const local070InstanceOptSep = defineInstance({
+  chain: localhost,
+  forkBlockNumber: 21153995,
+  forkUrl:
+    process.env.VITEST_OPT_SEPOLIA_FORK_URL ??
+    "https://opt-sepolia-rpc.publicnode.com",
+  entryPointVersion: "0.7.0",
+  anvilPort: 8345,
+  bundlerPort: 8445,
+});
```

Comment:
why do we need opt sepolia here?

Reply:
our temp deployments are only on opt sepo. we're in the process of salt mining to deploy more efficient ones, those will land next monday

Reply:
oh also this is gonna cause clashes with the other instances because it uses the same port as the `local070instance`. you should be fine to just use the existing 070 instance tbh

Reply:
oh I see, then you'll need to update the `setupTests` file as well to account for this instance. or just use the `sepolia` instance in this PR and on monday when we have deployments the tests will pass (I would opt for this so we don't have to manage so many instances in CI and locally)

Reply:
hmm since we need this for now, will keep this in + keep this comment up is till Monday/when contracts are deployed, then will remove this instance